pkg/schema/tracker.go

```diff
@@ -155,10 +175,29 @@ func NewTracker(ctx context.Context, task string, sessionCfg map[string]string,
 		return nil, err
 	}
 
+	// init downstreamTracker
+	downstreamTracker, err := initDownStreamTracker(sessionCfg["sql_mode"])
```

Comment:
what if "sql_mode" not in `sessionCfg `? BTW, you can use `utils.GetParser` to init parser it will auto adjust `sql_mode` for you

Reply:
Thank you for Function `NewTracker` gets "sql_mode" from stmt "SHOW VARIABLES" which is same as `utils.GetParser`,  I'm afraid it will get "sql_mode" from db again, so i use  `utils.GetParserFromSQLModeStr`.